### ISSUE TITLE
UAT: "Dresses" link results in 500 error

### ISSUE THREAD

**Issue by awilcoxa** [magento-research/pwa-studio#690]
The "Dresses" link in the main nav drawer (left) resolves to a 500 internal server error. It should point to the same URL as the category page navigation for "Dresses"

Issues #6: https://wiki.corp.magento.com/display/FF/Venia+UX+Issues 

## This issue is for the following packages:

- [x ] `venia-concept`
- [ ] `pwa-buildpack`
- [ ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [x ] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

**Comment by awilcoxa** [comment]
In checking this, I'm not seeing the same behavior. Requires additional investigation.

**Comment by Starotitorov** [comment]
@awilcoxa, I can not reproduce the issue.

**Comment by awilcoxa** [comment]
@dmtrkad @Starotitorov Yes, as indicated in my 1st comment, I wasn't able to reproduce this either. It was something that @soumya-ashok ran across but may no longer be an issue. If no one can reproduce then we should simply close this as "invalid"

**Comment by dmtrkad** [comment]
@soumya-ashok , in case this issue is not reproducible for you, please update it with the status or provide steps to get specified the actual result into the desciption.
